packages/fx-core/resource/package.nls.json

```diff
@@ -127,14 +127,14 @@
-  "core.migrationV3.Message": "The Teams Toolkit related configurations are older version. Upgrading project is required to continue. Please confirm you are agree to upgrade the TeamsFx configurations.\nThe upgrading will not impact your business source code, but just the configuration used by Teams Toolkit, and backups will be generated for reverting. If you don't agree to upgrade, you can download previouse version %s of Teams Toolkit to continue working with current config. Click Learn More to get more information.",
+  "core.localDebug.tenantConfirmNotice": "You have already run debug for the Teams app in another tenant '%s'. You can sign out of '%s' and sign in with the matching Microsoft 365 account to re-use the previous debug resources. If you want to switch tenants for debugging, delete the local file(s) (%s) and start debugging again.",
+  "core.localDebug.tenantConfirmNoticeWhenAllowSwitchAccount": "You already run debug for the Teams app in another tenant '%s'. If you want to switch tenants for debugging, delete the local file(s) (%s) and start debugging again.",
+  "core.migrationToArmAndMultiEnv.Message": "Teams Toolkit will upgrade your project configuration files to support the latest features. The upgrade process will not change your custom code and create the backup files in case a revert is needed.\nNotice this upgrade is a must-have to continue to use current version Teams Toolkit. If you are not ready to upgrade and want to continue using the old version, please install Teams Toolkit version <= 2.10.0",
+  "core.migrationV3.Message": "To continue, an upgrade to the Teams Toolkit configuration files is required and can be done automatically. Or, you can install Teams Toolkit %s to continue using this project.\n\nSelect Upgrade to perform the automatic upgrade. A backup of the configuration files will be created in case you need to revert.",
   "core.migrationV3.VS.Message": "The installed version of Teams Toolkit (%s) requires configuration changes to this project that can be done automatically. Before upgrading, select Learn more to read about the required changes or select Upgrade to perform the changes automatically.",
-  "core.migrationToArmAndMultiEnv.SuccessMessage": "Upgrade succeeded. Your project is upgraded to new configuration files.",
-  "core.migrationToArmAndMultiEnv.ErrorMessage": "Upgrade Failed. Please check the error message in Output window to troubleshoot and fix. You can also click Learn More for the FAQ and how to upgrade manually.",
-  "core.migrationToArmAndMultiEnv.PreCheckErrorMessage": "Upgrade Failed. '%s' doesn't exist or is not in json format. Please fix it and try again by running command (Teams: Upgrade project).\nYou can also click Learn More for the FAQ.",
+  "core.migrationToArmAndMultiEnv.SuccessMessage": "Upgrade succeeded. Your project was upgraded to the new configuration files.",
```

Comment:
I noticed that some of these have the second word capitalized (the "Failed" state ones) while others have the second word lowercase. We should stardardize on one way to do that.

Reply:
Done!